CMakeLists.txt

```diff
@@ -244,7 +244,9 @@ endif()
 # Dependencies version, this must match the number in external_deps/build.sh
 set(DEPS_VERSION 10)
 
-if (NOT NACL)
+option(USE_EXTERNAL_DEPS "Download build dependencies, required for building an running NaCl nexe binaries." ON)
```

Comment:
I find the description not totally accurate as it not only controls downloading, but also whether dependencies that are already there (which may even have been built by the user). How about "Download or reuse dependencies from EXTERNAL_DEPS_DIR (mandatory for building and running NaCl .nexe binaries)"

Reply:
Done.